introduction.md

```diff
@@ -1,2 +1,10 @@
 This is my very interesting introduction.
 My favorite Code School path is the JavaScript path.
+Deadly skills:
+* Explosive stare
+* Laser fingers
+* Swordfighting
+* Rocket launcher arms
+* Mind control
+* Daedly acurate spelling (or the other way aruond)
+* Daydreaming
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?